### DISCUSSION TITLE
Incorporate/merge with nougat from facebookresearch?

### DISCUSSION THREAD

**Discussion by universemaster** [lukas-blecher/LaTeX-OCR#311] [General]
[https://github.com/facebookresearch/nougat](nougat) is new from facebook research.

My initial tests show impressive results.  See [facebookresearch.github.io/nougat/](https://facebookresearch.github.io/nougat/)

**Comment by lukas-blecher** [comment]
Thank you for your suggestion. I was already aware of nougat, since I'm the first author 😅

I think it would make a good addition, maybe as an optional plug-in.

**Reply by rainyl** [reply]
Great！Congratulations！
